@codexteam/ui/dev/pages/components/Chart.vue

```diff
@@ -0,0 +1,450 @@
+    };
+  }
+});
+
+/* ==================== Multiple Lines Chart ==================== */
+
+/**
+ * Currently selected range option for multiple lines chart
+ */
+const multiRangeSelected = ref<DefaultItem>({
+  title: 'Week',
+  onActivate: onActivateOptionSelect,
+});
+
+/**
+ * Currently selected grouping option for multiple lines chart
+ */
+const multiGroupingSelected = ref<DefaultItem>({
+  title: 'Days',
+  onActivate: onActivateOptionSelect,
+});
+
+/**
+ * Available range options for multiple lines chart
+ */
+const multiRangeItems: ContextMenuItem[] = [
+  {
+    title: 'Hour',
+    onActivate: onActivateOptionSelect,
+  },
+  {
+    title: 'Day',
+    onActivate: onActivateOptionSelect,
+  },
+  {
+    title: 'Week',
+    onActivate: onActivateOptionSelect,
+  },
+  {
+    title: 'Month',
+    onActivate: onActivateOptionSelect,
+  },
+];
+
+/**
+ * Computed range value derived from selected range title
+ */
+const multiChartRange = computed<RangeValue>(() => {
+  return rangeTitleToValue[multiRangeSelected.value.title] || 'week';
+});
+
+/**
+ * Computed grouping value derived from selected grouping title
+ */
+const multiChartGrouping = computed<GroupingValue>(() => {
+  return groupingTitleToValue[multiGroupingSelected.value.title] || 'days';
+});
+
+/**
+ * Available grouping options filtered by current range
+ * - month → only Days
+ * - week → Days, Hours
+ * - day → Hours, Minutes
+ * - hour → only Minutes
+ */
+const multiGroupingItems = computed<ContextMenuItem[]>(() => {
+  const range = multiChartRange.value;
+
+  return [
+    {
+      title: 'Minutes',
+      onActivate: onActivateOptionSelect,
+    },
+    {
+      title: 'Hours',
+      onActivate: onActivateOptionSelect,
+    },
+    {
+      title: 'Days',
+      onActivate: onActivateOptionSelect,
+    },
+  ].filter(item => isGroupingAvailable(range, groupingTitleToValue[item.title]));
+});
+
+/**
+ * Chart data for multiple lines chart with two lines: accepted and rate-limited
+ */
+const multipleLinesData = computed<ChartLine[]>(() => [
+  {
+    label: 'accepted',
+    data: generateData(multiChartRange.value, multiChartGrouping.value, 150),
+    color: ChartLineColor.Red,
+  },
+  {
+    label: 'rate-limited',
+    data: generateData(multiChartRange.value, multiChartGrouping.value, 50),
+    color: ChartLineColor.LightGrey,
+  },
+]);
+
+/**
+ * Auto-set grouping to default when range changes and current grouping is not available
+ */
+watch(multiChartRange, (newRange) => {
+  const currentGrouping = multiChartGrouping.value;
+
+  if (!isGroupingAvailable(newRange, currentGrouping)) {
+    multiGroupingSelected.value = {
+      title: getDefaultGroupingTitle(newRange),
+      onActivate: onActivateOptionSelect,
+    };
+  }
+});
+</script>
+
+<style scoped>
+.charts {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: var(--spacing-l);
+  margin-bottom: var(--spacing-xl);
+
+  &__showcase {
+    position: relative;
+    width: 100%;
+    height: 250px;
```

Comment:
lets remove hardcoded height to get rid of extra space below the chart

<img width="976" height="315" alt="Image" src="https://github.com/user-attachments/assets/f38c37b5-61fb-45a1-a0fd-9c498b3ead03" />